### ISSUE TITLE
After I install Ciphey, I run the command “Ciphey ”.The system return an error.

### ISSUE THREAD

**Issue by PureWaterSun** [Ciphey/Ciphey#323]
**Describe the bug**
After I install Ciphey, I run the command “Ciphey ”.The system return an error.

=================

192:~ hh$ ciphey 
Traceback (most recent call last):
  File "/usr/local/bin/ciphey", line 5, in <module>
    from ciphey.ciphey import main
  File "/usr/local/lib/python3.7/site-packages/ciphey/__init__.py", line 1, in <module>
    from . import common
  File "/usr/local/lib/python3.7/site-packages/ciphey/common.py", line 5, in <module>
    import cipheycore
  File "/usr/local/lib/python3.7/site-packages/cipheycore/__init__.py", line 13, in <module>
    from . import _ciphey_core_py
ImportError: dlopen(/usr/local/lib/python3.7/site-packages/cipheycore/_ciphey_core_py.so, 2): Library not loaded: /usr/local/opt/python@3.7/Frameworks/Python.framework/Versions/3.7/Python
  Referenced from: /usr/local/lib/python3.7/site-packages/cipheycore/_ciphey_core_py.so
  Reason: image not found

My environment

**Version**
Name: ciphey
Version: 5.3.2
Summary: Automated Decryption Tool
Home-page: None
Author: Brandon
Author-email: brandon@skerritt.blog
License: MIT
Location: /usr/local/lib/python3.7/site-packages
Requires: base58, pyyaml, click, appdirs, yaspin, click-spinner, rich, typing-inspect, base91, cipheycore, pybase62, flake8, pylint, loguru, cipheydists
Required-by: 
---
Name: cipheycore
Version: 0.2.9
Summary: 
Home-page: None
Author: Cyclic3
Author-email: cyclic3.git@gmail.com
License: None
Location: /usr/local/lib/python3.7/site-packages
Requires: 
Required-by: ciphey
---
Name: cipheydists
Version: 0.3.22
Summary: A collection of distributions, character sets and dictionaries for use in ciphey
Home-page: None
Author: Cyclic3
Author-email: cyclic3.git@gmail.com
License: None
Location: /usr/local/lib/python3.7/site-packages
Requires: 
Required-by: ciphey

I don't know how to solve this problem, please help me！！！

**Comment by brandonskerritt** [comment]
Per the bug report template, we have this section:
https://github.com/Ciphey/Ciphey/wiki/Common-Issues-&-Their-Solutions
If you had filled out the bug report, you would have seen we have issues with Mac OS which are solved by using Docker.

This GitHub issue does not follow our template, it is a duplicate of another issue, and this issue is solved in our documentation so I am closing it :)
